### PR TITLE
Notebook for PDC to CRDCH transformation workflow

### PR DESCRIPTION
**Pull request by sujaypatil96**
Similar to the [GDC to CCDH conversion](https://github.com/cancerDHC/example-data/blob/main/GDC%20to%20CCDH%20conversion.ipynb) notebook tutorial we have, this PR seeks to demonstrate the same with PDC data.

Note: The data in GDC and PDC is similar for the most part, so most of the snippets were replicated as is.

This PR is by no means complete. A few things that need to be added:

- [x] Replace the dataclass file being used currently with the latest dataclass file as in the [PyPI package](https://pypi.org/project/crdch-model/)
- [x] Add more documentation

CC: @gaurav @turbomam @jooho-lee-kim. Just wanted to get your eyes on the Notebook is all.

### CODE REVIEW

**Comment by turbomam** [comment]
Thanks for starting this @sujaypatil96 

You know I get confused easily, so let me know if I'm rehashing something we've already resolved!

Your `pdc_to_crdch_transformation` notebook imports the CRDC-H model with 

`from ccdh import ccdhmodel as ccdh`

That gets an old version of the model from the cloned repo in the local filesystem.

In order to develop against the latest model as published on PyPI, I would suggest

`import crdch_model.crdch_model as ccdh`

In which case you will get the following from `create_stage_from_pdc ()`

> TypeError: crdch_model.crdch_model.CodeableConcept() argument after ** must be a mapping, not str

**Comment by gaurav** [comment]
> That gets an old version of the model from the cloned repo in the local filesystem.

Yeah, it's a little odd that we still have that version rattling around! I'm working on a PR to switch this repo over to Poetry and to delete those files from this repo entirely (https://github.com/cancerDHC/example-data/pull/41).

**Comment by sujaypatil96** [comment]
I think this PR is ready for review @turbomam and @gaurav. Would love to hear some thoughts from you guys as to how I can further improve this tutorial.

File: pdc_to_crdch_transformation.ipynb [hunk 1]
```diff
@@ -0,0 +1,879 @@
...
+    "    \"\"\"Create a CCDH BodySite based on the name of a site in the human body.\"\"\"\n",
+    "    \n",
+    "    # Accept 'None'.\n",
+    "    if site_name is None:\n",
+    "        return None\n",
+    "    \n",
+    "    # Some body sites are not currently included in the CCDH model. We will need to translate these sites\n",
+    "    # into values that *are* included in the CCDH model.\n",
+    "    site_mappings = {\n",
+    "        'Larynx, NOS': crcdh_model.EnumCRDCHBodySiteSite.Larynx\n",
```
Comment (turbomam) [inline pdc_to_crdch_transformation.ipynb:785]: shouldn't that be crdch_model, not crcdh_model?

**Review by turbomam — CHANGES_REQUESTED** [review]

File: pdc_to_crdch_transformation.ipynb (hunk 1 above)
Comment (turbomam) [reply pdc_to_crdch_transformation.ipynb:785]: I did a global find and replace and the notebook ran to completion

Comment (sujaypatil96) [reply pdc_to_crdch_transformation.ipynb]: Great catch @turbomam. Thank you. I've changed the typo to the right spelling.

**Pull request merged**